source/regex.tex

```diff
@@ -3916,7 +3916,7 @@
 sequence \end{note}
 
 \pnum
-The \textit{current match} is \tcode{(*position).prefix()} if \tcode{subs[N] == -1}, or
+The \term{current match} is \tcode{(*position).prefix()} if \tcode{subs[N] == -1}, or
 \tcode{(*position)[subs[N]]} for any other value of \tcode{subs[N]}.
 
 \rSec3[re.tokiter.cnstr]{\tcode{regex_token_iterator} constructors}
```

Comment:
This patch adds \term, which this issue tries to eradicate (admittedly morphed from even worse \textit). Please drop this patch from here for now; I expect those \textit to mostly become \defn or \defnx.